network_analysis/find-service-areas-for-multiple-facilities/README.md

```diff
@@ -0,0 +1,38 @@
+# Find service areas for multiple facilities
+
+Find the service areas of several facilities from a feature service.
+
+![](FindServiceAreasForMultipleFacilities.png)
+
+## Use case
+
+A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.
```

Comment:
Is cost a supported factor?

Reply:
I think your second sentence gets to the heart of it. Like I've suggested before, try to write these so it goes straight to the example:

A city taxi company may calculate service areas around their vehicle lots to identify gaps in coverage. Alternatively, they may want to ensure overlaps where a high volume of passengers requires redundant facilities, such as near an airport.